obspy/taup/tests/test_tau.py

```diff
@@ -172,82 +180,96 @@ def test_p_iasp91_geo_fallback_manual(self):
 
     def test_p_iasp91(self):
         """
-        Test P phase arrival against TauP output in in model IASP91.
+        Test P phase arrival against TauP output in in model IASP91
```

Comment:
Not sure that all of these docstrings needed to change, but if you're going to do so, can you remove the duplicate 'in in'?